ygnmi/ygnmi.go

```diff
@@ -198,6 +211,12 @@ func Lookup[T any](ctx context.Context, c *Client, q SingletonQuery[T], opts ...
 	return val, nil
 }
 
+// LookupConfig fetches the value of a ConfigQuery with a ONCE subscription.
+// Note: This is a workaround for Go's type inference not working for this use case and may be removed in a subsequent release.
```

Comment:
I'd probably add a line that it is equivalent to calling Lookup and providing a fully-qualified type parameter. Same below.

Reply:
going to do this in ondatra instead